plugins/single_plugins/switcher.cpp

```diff
@@ -126,6 +126,9 @@ class WayfireSwitcher : public wf::plugin_interface_t
         };
 
         touch_activate = [=] (wf_touch_gesture*) {
+            if (!output->activate_plugin(grab_interface))
```

Comment:
This is wrong, the plugin is activated only in the `!active` case, check the code of `handle_switch_request`

Reply:
I pushed an update for this, please check if it's doing The Right Thing.